apps/andi/lib/andi_web/live/ingestion_live_view/transformations/transformation_field_builder.ex

```diff
@@ -0,0 +1,25 @@
+defmodule AndiWeb.IngestionLiveView.Transformations.TransformationFieldBuilder do
+  import Phoenix.HTML.Form
+  import Phoenix.LiveView.Helpers
+
+  def build_input(%{field_type: "string"} = field, assigns, form) do
```

Comment:
Should there be a default case for this for error handling?

Reply:
Probably just in case. I'll put one in on the next card.